src/test/java/guitests/MenuControlTest.java

```diff
@@ -21,6 +22,8 @@ public void menuControlTest() {
         modelUpdatedEventTriggered = false;
         UI.events.registerEvent((ModelUpdatedEventHandler) e -> modelUpdatedEventTriggered = true);
         PanelControl panelControl = (PanelControl) find("#dummy/dummy_col0").getParent();
+        Preferences testPref = panelControl.getPreferences();
```

Comment:
Right, `prefs` isn't static. Moving it to `UI` would still require a reference to the current `UI` instance though. How about making `prefs` static, then doing that?